### DISCUSSION TITLE
C drive getting full while creating images and containers ?

### DISCUSSION THREAD

**Discussion by bathivinod** [containers/podman#19464] [General]
I am using Podman Desktop in windows 11, Now when ever I create any images or containers my `C` Drive getting full and out of space. Is there a way to change the directory from `C` Drive to another drive in windows 11 ?

**Comment by bathivinod** [comment]
Hi Team,
Any suggestions on above query.

**Comment by rhatdan** [comment]
Why are you asking aboud Docker Desktop at Podman site?

**Comment by bathivinod** [comment]
Hi @rhatdan 
Sorry, it's my bad, its a typo, Instead of mentioning `Podman Desktop` , I mentioned Docker Desktop

**Reply by rhatdan** [reply]
Is the system using vfs driver?  You should be able to see this with podman info.

**Comment by rhatdan** [comment]
@n1hility @benoitf FYI

**Comment by n1hility** [comment]
I think this feature request is probably what you are looking for: containers/podman#13984 ? An ability to specify the vhdx file somewhere else (in this case off of the C drive).  

Note in that issue is some instructions on how you can do this manually until the feature lands

**Comment by bathivinod** [comment]
Yah similar to that but we are using wsl instead on vhd

**Reply by n1hility** [reply]
WSL internally uses a vhd file for the storage volume, so contains all container storage state

**Reply by bathivinod** [reply]
so, if we move WSL from C: drive to another drive eg: E: drive will it be any impact on Podman ?
